### PR TITLE
Undo accidental commit

### PR DESCRIPTION
**Pull request by rando2**
### Description of the proposed additions or changes

I accidentally committed to master while trying to fix a merge conflict on @aimundo's fork! This just undoes that commit.

### Checklist

- [ ] Text is formatted so that each sentence is on its own line.

- [ ] Pre-prints cited in this pull request have a GitHub issue opened so that they can be reviewed.

### CODE REVIEW

**Comment by agitter** [comment]
What's the desired net change?  Currently the Files Changed tab shows no modifications.  Did the merge from master in 1888dd1 undo some of the updates?

**Comment by rando2** [comment]
@agitter It must have, sorry about that -- I thought it was just because GitHub has been super laggy for me today. The diff is clean now (at least what is loading for me!)

File: content/metadata.yaml [hunk 1]
```diff
@@ -1397,12 +1397,8 @@ authors:
...
     github: aimundo
     name: Ariel I. Mundo
-        affiliations:
-          - institution: University of Arkansas
-            department: Department of Biomedical Engineering
-            city: Fayetteville
-            state: AR
-            country: USA
+    affiliations:
+    - Department of Biomedical Engineering, University of Arkansas, Fayetteville, Arkansas, USA
```
Comment (agitter) [inline content/metadata.yaml:1401]: The spacing is different here, but the AppVeyor artifact looks okay so it didn't matter.

**Review by agitter — APPROVED** [review]
Reverts d914cdec15975c2b0ee67fbecef44eb0e40dcff9

**Pull request merged**